auditlog/migrations/0008_auto_20210826_1434.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.6 on 2021-08-26 12:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("auditlog", "0007_object_pk_type"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="logentry",
+            name="action",
+            field=models.PositiveSmallIntegerField(
+                choices=[(0, "create"), (1, "update"), (2, "delete"), (5, "access")],
+                verbose_name="action",
+            ),
+        ),
+        migrations.AlterField(
```

Comment:
Do we need this migration?

Reply:
See below, I will recreate the migrations.